packages/types/src/index.ts

```diff
@@ -319,18 +327,8 @@ export type Leg = {
   routeTextColor?: string;
   routeType?: number;
   serviceDate?: string;
-  startTime: number;
+  startTime: number | string;
```

Comment:
Why is this possibly a string? In the API response it is just a long. 
<img width="383" alt="image" src="https://github.com/opentripplanner/otp-ui/assets/92878779/69950220-eab2-4fcd-8eeb-d21b7ef6d6f7">

Reply:
TNC legs can be a string I believe